src/test/Unit.test.js

```diff
@@ -20,7 +25,7 @@ describe('Route unit tests', () => {
 
             describe('RouteUtils', () => {
                 test('RouteUtils.getGraphhopperBusParams', () => {
-                    const o = RouteUtils.getGraphhopperBusParams(end, start, time, arriveBy);
+                    const o = GhopperUtils.getGraphhopperBusParams(end, start, time, arriveBy);
```

Comment:
not a fan of this variable naming, what is `o`

Reply:
This is more readable, and it's just a test of a getter method, so naming is relatively unimportant.
![image](https://user-images.githubusercontent.com/7935434/47475676-96601b00-d7ea-11e8-8be5-5a94621aa075.png)
![image](https://user-images.githubusercontent.com/7935434/47475680-9e1fbf80-d7ea-11e8-8faf-ac493d491996.png)